Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ghcr.io/imageomics/dataverse-access:0.0.3 as model_fetcher
```

Comment:
I'm wondering whether pinning this image at the current version is really a good idea. It means that if we were to update the image as a result of a bug fix or code improvement, we'd have to update the repo here too.

OTOH, we also don't want to risk updating to an image version that is no longer compatible with the behavior expectations codified here.

Perhaps there's some middle ground by promoting `dataverse-access` to 0.1.0, and releasing every 0.1.x image also as version 0.1, which we would pin here. Thoughts?

Reply:
Currently the github automation for dataverse-access only tags with `{{major}}.{{minor}}.{{patch}}`. 
Tagging with `{{major}}.{{minor}}` and `{{major}}` should be possible and would be a nice improvement.
Perhaps we should go ahead and create version `1.0.0` since in semver:
> [Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.](https://semver.org/#spec-item-4).

Reply:
I switched to using the `1` release of `dataverse-access` so rebuilding the container from this repo will receive bug fixes.